### PR TITLE
add doc for build in docker

### PR DESCRIPTION
**Pull request by acmol**

### CODE REVIEW

File: doc/build.rst [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
-mkdir -p ../build && cd ../build && cmake ..
+    cd bigflow_python/python/
+    sh run-tests
+    
+    
+使用Docker构建
+""""""""""""""""""""""
 
-make
+运行`docker run -i -t -v $PWD:/root/bigflow centos:7.1.1503 /bin/bash`进入centos docker镜像
```
Comment (advancedxy) [inline doc/build.rst]: Where's download docker image step?

File: doc/build.rst [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
 
-make release
+在镜像中执行：::
+  
+    yum install sudo -y
```
Comment (advancedxy) [inline doc/build.rst:37]: where is the git clone step?

If you assume that user already clones bigflow's repo, then make it clear in the comment.

**Review by advancedxy — CHANGES_REQUESTED** [review]

File: doc/build.rst (hunk 1 above)
Comment (acmol) [reply doc/build.rst]: It can automatically download the image from the official repo if there is no local image.

File: doc/build.rst (hunk 2 above)
Comment (acmol) [reply doc/build.rst:37]: You said it. My fault.

**Review by advancedxy — APPROVED** [review]
LGTM.

We may add a note about MAKEFLAGS env

**Pull request merged**